scripts/example_script.py

```diff
@@ -0,0 +1,241 @@
+import girder_client
```

Comment:
Let's put the standard Python file encoding and copyright comments at the top. Look at any Python file in Girder to grab it.